flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/types/SlotID.java

```diff
@@ -75,9 +75,15 @@ public int hashCode() {
 
 	@Override
 	public String toString() {
-		return "SlotID{" +
-			"resourceId=" + resourceId +
-			", slotId=" + slotId +
-			'}';
+		return resourceId + "_" + slotId;
+	}
+
+	/**
+	 * Generate a random slot id.
+	 *
+	 * @return A random slot id.
+	 */
+	public static SlotID generate() {
+		return new SlotID(ResourceID.generate(), 0);
```

Comment:
This always generates the same slot id. At this point I'm not really sure what the purpose of the slot id is.